module/platform/NUSense/HardwareIO/README.md

```diff
@@ -21,3 +21,49 @@ This module is built when the subcontroller CMake flag is set to `NUSense`. Usin
 - `message::input::RawSensors` containing the current NUgus sensor data from the NUSense device.
 
 ## Dependencies
+
+## Notes
+
+There are 2 ways to generate nanopb messages for NUSense. The generated files must then go into the NUSense NUController repo before building the binaries onto the board. This will be a temporary solution until a `./b install` pipeline for NUSense is set up. The guides below will assume that the OS is some linux based distro.
+
+### b script
+
+For example, if a user wants to generate nanopb messages and specify that certain properties of messages must have max counts, the command below should put the generated files in the `recordings` directory.
+
+```bash
+./b nusense generate_nanopb_message recordings/ServoTarget.proto \
+    --selections ServoTargets:targets:max_count:20 \
+                 SubcontrollerServoTargets:targets:max_count:20
+```
+
+The `recordings` directory is convenient as it is symlinked to the docker container. It is also imperative to specify maximum counts for repeated fields as encoding a message without it takes significantly longer and introduces a lot of overhead that is undesired in terms of performance.
```

Comment:
Don't do this as a b script. Integrate into the build pipeline of `nuclear_message` so that the generated files are automagically updated whenever the proto files or options files are updated. Also just commit the options files next to the proto files, these can then be dependencies of the build process as well

This also means that when you get around to the `./b install` side of things you wont need to do anything special

Reply:
I will address this in a different pr, I will be using this tool a lot later on for a task. I will delete it after I've finished said task.